src/components/movement-form/index.tsx

```diff
@@ -279,6 +282,8 @@ export default function MovementForm({
           <GridItem colSpan={2}>
             <Input
               label="Responsável"
+              readOnly
+              defaultValue={user?.name}
```

Comment:
Seria bom deixar os campos como desabilitados.
```suggestion
              defaultValue={user?.name}
              disabled
```

Reply:
se deixar como disabled o programa não ta pegando o conteúdo da box, já quando eu deixo readonly funciona